apps/whale-api/src/module.api/poolpair.controller.e2e.ts

```diff
@@ -368,12 +368,14 @@ describe('get best path', () => {
           poolPairId: '13',
           priceRatio: { ab: '0.50000000', ba: '2.00000000' },
           tokenA: { id: '1', symbol: 'A', displaySymbol: 'dA' },
-          tokenB: { id: '0', symbol: 'DFI', displaySymbol: 'DFI' }
+          tokenB: { id: '0', symbol: 'DFI', displaySymbol: 'DFI' },
+          estimatedReturn: '2.00000000',
+          estimatedDexFees: []
         }
       ],
       estimatedReturn: '2.00000000'
     })
-    expect(paths1.bestPath).toStrictEqual(paths2.bestPath)
+    // expect(paths1.bestPath).toStrictEqual(paths2.bestPath)
```

Comment:
hmm.. shouldn't the `paths2` contain the estimated value?? :thinking:

Reply:
`paths1.bestPath[0].estimatedReturn` value is denoted in DFI
`paths2.bestPath[0].estimatedReturn` value is denoted in A

I think we can delete this test - it might not always be the case that the best path from a->b will be the same as b->a

Reply:
Updated (removed)!